spec/bookshelf-spec.ts

```diff
@@ -616,6 +616,48 @@ describe('Bookshelf links', () => {
     expect(_.matches(expected)(result)).toBe(true);
   });
 
+  it('should not add pagination links if collection is empty', () => {
+    let limit: number = 10;
+    let offset: number = 40;
```

Comment:
It just seems weird that `offset > total`, but you are right that it should work nonetheless!